### ISSUE TITLE
bcftools csq: Change chromosome name requirement.

### ISSUE THREAD

**Issue by niederhuth** [samtools/bcftools#2294]
Could you please drop the requirement that chromosome start with "chr" from bcftools csq. This creates many headaches when working with other species where this is not the case.

**Comment by pd3** [comment]
I don't think this is required, the program should respect whatever is in the VCF, GFF and Fasta file. If it's not working with your data, please provide a small test case to reproduce the problem. Also, make sure you are using the latest version, there were some improvements in the code to work around situations when the three input files used different conventions.
